docs/core/tools/dotnet-run.md

```diff
@@ -74,213 +46,85 @@ To run the application, the `dotnet run` command resolves the dependencies of th
 
-Delimits arguments to `dotnet run` from arguments for the application being run. All arguments after this delimiter are passed to the application run.
+  Forces all dependencies to be resolved even if the last restore was successful. Specifying this flag is the same as deleting the *project.assets.json* file. Available since .NET Core 2.0 SDK.
 
-`-c|--configuration {Debug|Release}`
+- **`-h|--help`**
 
-Defines the build configuration. The default value for most projects is `Debug`.
+  Prints out a short help for the command.
 
-`-f|--framework <FRAMEWORK>`
+- **`--interactive`**
 
-Builds and runs the app using the specified [framework](../../standard/frameworks.md). The framework must be specified in the project file.
+  Allows the command to stop and wait for user input or action (for example, to complete authentication). Available since .NET Core 3.0 SDK.
 
-`--force`
+- **`--launch-profile <NAME>`**
 
-Forces all dependencies to be resolved even if the last restore was successful. Specifying this flag is the same as deleting the *project.assets.json* file.
+  The name of the launch profile (if any) to use when launching the application. Launch profiles are defined in the *launchSettings.json* file and are typically called `Development`, `Staging`, and `Production`. For more information, see [Working with multiple environments](/aspnet/core/fundamentals/environments). Available since .NET Core 2.0 SDK.
 
-`-h|--help`
+- **`--no-build`**
 
-Prints out a short help for the command.
+  Doesn't build the project before running. It also implicit sets the `--no-restore` flag. Available since .NET Core 2.0 SDK.
 
-`--launch-profile <NAME>`
+- **`--no-dependencies`**
 
-The name of the launch profile (if any) to use when launching the application. Launch profiles are defined in the *launchSettings.json* file and are typically called `Development`,
-`Staging`, and `Production`. For more information, see [Working with multiple environments](/aspnet/core/fundamentals/environments).
+  When restoring a project with project-to-project (P2P) references, restores the root project and not the references. Available since .NET Core 2.0 SDK.
 
-`--no-build`
+- **`--no-launch-profile`**
 
-Doesn't build the project before running. It also implicit sets the `--no-restore` flag.
+  Doesn't try to use *launchSettings.json* to configure the application. Available since .NET Core 2.0 SDK.
 
-`--no-dependencies`
+- **`--no-restore`**
 
-When restoring a project with project-to-project (P2P) references, restores the root project and not the references.
+  Doesn't execute an implicit restore when running the command. Available since .NET Core 2.0 SDK.
 
-`--no-launch-profile`
+- **`-p|--project <PATH>`**
 
-Doesn't try to use *launchSettings.json* to configure the application.
+  Specifies the path of the project file to run (folder name or full path). If not specified, it defaults to the current directory.
 
-`--no-restore`
+- **`--r|runtime <RUNTIME_IDENTIFIER>`**
```

Comment:
Was the short option made available at a later release only? Or was it just a --help issue?

Reply:
@dsplaisted correct me if I'm wrong, but it looks like it's new with 3.0.100.

Reply:
@wli3 to comment on the short version of the RID parameter.  I think the short version has been around since the beginning, but only uses one dash.

Reply:
i need to dig the code

Reply:
It's a 3.0 thing https://github.com/dotnet/cli/commit/3dd7968d963cd21e9e8626bce69b76fa48cff05b